.cursor/plans/integr-aad39d-b6fbec35.plan.md

```diff
@@ -0,0 +1,126 @@
+- Remove the external Storybook link.
+
+5) Versioning setup (per-plugin)
+
+- Enable versioning for the `web-components` plugin instance in `docusaurus.config.ts`.
+- Create `wc-docs/versions.json` on first versioning.
+- Add a helper script `scripts/version-web-components.mjs` that runs `docusaurus docs:version web-components <version>` (uses plugin id) so we can cut a docs version that corresponds to the library release (e.g., `6.4`).
+- The navbar will show a version switcher tied specifically to the `web-components` docs.
+
+6) CI automation (recommended)
+
+- New GitHub Action in this repo:
+- Triggers:
+- On schedule (e.g., daily) and
+- On `repository_dispatch` or on release/tag from `web-component-library`.
+- Steps: checkout → run `pnpm install` → `pnpm sync:webcomponents` → open PR if changes detected.
+- Optional: when a tag matching `@justifi/webcomponents@x.y.z` is released, run `scripts/version-web-components.mjs x.y` then sync and open PR.
+
+## Minimal changes required in web-component-library (high-level)
+
+1) File placement and naming
+
+- For each component: `packages/webcomponents/src/components/<Component>/docs/`
+- `index.mdx` (component overview)
+- `examples/*.mdx` (optional)
+- `assets/*` (images) → use relative paths `./assets/...`
+- General docs: `packages/webcomponents/src/docs/`
+- `overview/intro.mdx`, `guides/*.mdx`
+
+2) Frontmatter conventions (to play well with Docusaurus)
+
+- Each MD/MDX page should include:
+- `title`, `description`
+- optional `sidebar_label`, `slug`
+- Avoid repo-local absolute links; prefer relative links within each `docs` folder.
+
+3) MDX scope and components
+
+- Keep content as Markdown/MDX without Storybook-specific imports.
+- Use standard MDX/Docusaurus features: admonitions (`:::tip`), code blocks, tabs (`@theme/Tabs`).
+
+4) Asset and link hygiene
+
+- Reference images as `./assets/<file>`; the sync script will relocate and rewrite to `/img/web-components/...` in this site.
+- For cross-component links, prefer relative doc links; our sync script preserves folder structure under `wc-docs/components/`.
+
+5) Version alignment
+
+- Decide the mapping from package versions to docs versions (e.g., `6.4.x` → `6.4`).
+- On publish/release, emit a GitHub Action that calls this site’s `repository_dispatch` with `{version: '6.4'}` to cut docs version.
+
+## Notes on alternatives
+
+- Instead of API-based sync, we could use a git submodule (`vendor/web-component-library/`) and copy from the working tree during build. This adds `git submodule` maintenance but allows offline sync.
+- `git subtree` would bring a large history snapshot; API-based sync keeps this repo slim by importing only docs and assets.
+
+## Deliverables
+
+- A new `/web-components` section under the main navbar with its own sidebar and version switcher.
+- Automated sync from the source repo to keep docs current.
```

Comment:
Here - the syncing with the repo isn't what we want. The docs just need to install latest version of `@justifi/webcomponents` when a new version is released. Syncing with the repo directly would cause documentation of unreleased features and would be hard to version